### PR TITLE
PSI no calculation on `TextProfile`

### PR DESCRIPTION
**Pull request by taylorfturner**
don't return `PSI` on `TextColumn` profiling

### CODE REVIEW

**Review by tyfarnan — APPROVED** [review]

File: dataprofiler/tests/profilers/test_text_column_profile.py [hunk 1]
```diff
@@ -638,7 +637,6 @@ def test_diff(self):
                 "conservative": {"df": None, "p-value": None},
                 "welch": {"df": None, "p-value": None},
             },
-            "psi": 0,
```
Comment (JGSweets) [inline dataprofiler/tests/profilers/test_text_column_profile.py:641]: removing this, but this edge case needs testing elsewhere

**Review by JGSweets — CHANGES_REQUESTED** [review]
missing edge case test
